libs/types/src/tokens.rs

```diff
@@ -204,6 +204,48 @@ pub struct CustomMetadata {
 
 	/// Whether an asset can be used as a currency to fund Centrifuge Pools.
 	pub pool_currency: bool,
+
+	/// The ways this token is transferable from/to this chain
+	pub transferability: AssetTransferability,
+}
+
+#[derive(
+	Clone,
+	Copy,
+	Default,
+	PartialOrd,
+	Ord,
+	PartialEq,
+	Eq,
+	Debug,
+	Encode,
+	Decode,
+	TypeInfo,
+	MaxEncodedLen,
+)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum AssetTransferability {
+	/// The asset is only transferable through XCM
+	#[default]
+	Xcm,
+
+	/// The asset is only transferable through Centrifuge Connectors
+	Connectors,
+
+	/// The asset is transferable through all available options.
+	/// NOTE: Once set to `All`, the asset is automatically transferrable
+	/// through any eventual new option being added at a later stage. A
+	/// migration might be required if that's undesirable.
+	All,
+
+	/// The asset is not transferable
```

Comment:
```suggestion
	/// The asset is not cross-chain transferable
```

Reply:
With that, we could actually either:

- a. drop `None` and make it `Local` to say the asset is only transferrable locally (and not cross-chain) and make this the `#[default]` variant 
- b. Rename `AssetTransferability` to `AssetCrossChainTransferability`

I find that `b.` would make more sense because all fungible assets are locally transferable by nature so it seems like an unnecessary wide scope to cover; just don't like the longer name but maybe someone has a better suggestion? 😇